mkdocs/themes/readthedocs/base.html

```diff
@@ -39,7 +39,7 @@
     var mkdocs_page_url = {{ page.abs_url|tojson|safe }};
   </script>
   {% endif %}
-  <script src="{{ 'js/jquery-2.1.1.min.js'|url }}" defer></script>
+  <script src="{{ 'js/jquery-3.6.0.min.js'|url }}" defer></script>
```

Comment:
Generally the frontend follows the upstream ReadTheDocs theme (I think it was https://github.com/readthedocs/sphinx_rtd_theme but not sure). Have they updated JQuery as well?